### PR TITLE
link to API doc in index, add feature support matrix

### PR DESCRIPTION
**Pull request by ericxtang**
**What does this pull request do? Explain your changes. (required)**

**Specific updates (required)**

- Link to the API doc portion from the main doc page
- Add feature support matrix

**How did you test each of these updates (required)**

Tested by clicking and proof reading

**Does this pull request close any open issues?**

**Screenshots (optional):**

**Checklist:**

- [x] I have read the **CONTRIBUTING** document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/www/pages/docs/support-matrix.mdx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+| **Resolution** | **Supported tested**: custom resolution up to 1080p |
+| | **Supported untested**: custom resolution up to 4k|
+| **Pixel Format** | YUV4:2:0 |
+
+
+**Video Workflow Support**
+
+| Field   |  Notes      
+|----------| - | 
+| **Webhook Events** | `stream_started` |
```
Comment (darkdarkdragon) [inline packages/www/pages/docs/support-matrix.mdx]:
```suggestion
| **Webhook Events** | `streamStarted` |
```
it is called `streamStarted` in our API

File: packages/www/pages/docs/support-matrix.mdx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+| **Recording** | **Supported tested**: HLS playback with original renditions |
+| | **Unsupported**: mp4 download |
+| **Latency** | 10-18 seconds with good RTMP ingest health, 2s segment size, and standard 3 seg buffering in player |
+| **Output Format** | HLS playback URL (m3u8) |
+| **Video Players** | **Supported tested**: video.js, hls.js, safari/ios native player, Theoplayer |
+| | **Supported untested**: JWPlayer, android native, Shaka, Exo, Bitmovin, Brightcove |
+| **Video Delivery Location** | **Supported tested**: EU, US |
+| | **Supported untested**: Asia, South America, Australia, Africa |
+| | CDN delivery is ONLY available for paying users |
+| | Non-CDN delivery for free tier users is limited to 50mbps per stream, which roughly equals to 10 viewers in HD |
```
Comment (darkdarkdragon) [inline packages/www/pages/docs/support-matrix.mdx]: `50mbps per stream` - in our spec we have limit for number of viewers, not bandwidth

**Review by darkdarkdragon — CHANGES_REQUESTED** [review]

**Comment by ericxtang** [comment]
@darkdarkdragon updated based on your comment.

**Comment by darkdarkdragon** [comment]
@ericxtang 
`Non-CDN delivery for free tier users is limited to 50mbps per account, roughly equals to 10 viewers in HD. Additional viewers will automatically be served in 240p.`
This is no how we're planning this to work. It is like
`Non-CDN delivery for free tier users is limited to to 10 viewers per account. `

And somewhere in transcoding section:
`Free tier users will have transcoding limited to 240p after 30 free minutes`

**Comment by ericxtang** [comment]
> @ericxtang
> `Non-CDN delivery for free tier users is limited to 50mbps per account, roughly equals to 10 viewers in HD. Additional viewers will automatically be served in 240p.`
> This is no how we're planning this to work. It is like
> `Non-CDN delivery for free tier users is limited to to 10 viewers per account. `
> 
> And somewhere in transcoding section:
> `Free tier users will have transcoding limited to 240p after 30 free minutes`

Thanks @darkdarkdragon.  Updated based on this comment.

**Review by darkdarkdragon — APPROVED** [review]
LGTM

**Pull request merged**